src/overmap.cpp

```diff
@@ -1256,6 +1256,18 @@ const oter_id overmap::get_ter( const tripoint &p ) const
     return get_ter( p.x, p.y, p.z );
 }
 
+const std::string overmap::get_om_id( const int x, const int y, const int z ) const
+{
+    const oter_id &ter = get_ter( x, y, z );
```

Comment:
Why is this actually wrapped here? Why don't you do this outside of `overmap`, e.g. in the `game` functions write `some_overmap.get_ter( ... ).id()` instead of `some_overmap.get_om_id( ... )`?